doc/build-osx.md

```diff
@@ -141,6 +141,15 @@ brew install qt@5
 Note: Building with Qt binaries downloaded from the Qt website is not officially supported.
 See the notes in [#7714](https://github.com/bitcoin/bitcoin/issues/7714).
 
+Note: Be sure 'qt' (version 6.x) is not installed in brew together with 'qt@5'. Only 'qt@5' is required.
+If 'qt' (v6.x) is installed in macOS the build process will fail with a "unknown type name 'QStringRef'; did you mean 'CFStringRef'?" error. 
```

Comment:
```suggestion
If 'qt' is installed, the build process will fail with an error. 
```

Reply:
This is also tripping up the linter due to the trailing whitespace.